readme.md

```diff
@@ -132,7 +132,11 @@ In WP Admin, go to `Events`  > `Topics`.
 
 Topics are like post categories, but for events.
 
-### Get Collaborator Access
+## Contribute
+
+If you wish to share in the collaborative of work to build _GatherPress_, please drop us a line either via [WordPress Slack](https://make.wordpress.org/chat/) or on [GatherPress.org](htps://gatherpress.org/get-involved). The development location of the GatherPress project can be found at [https://github.com/gatherpress/gatherpress](https://github.com/gatherpress/gatherpress). All contributions are welcome: code, design, user interface, documentation, translation, and more.
+
```

Comment:
This could be a separate PR since it appears to be unrelated to the OSM work.

Reply:
On line 90 of the readme.md it has coming soon for OSM. Let's remove that since this PR adds OSM.

Reply:
Need to add leaflet and react-leaflet to 3rd party libraries we are using in readme too

Reply:
resolved! marking conversation as resolved at this time.

Reply:
I'd still move this `Contribute` piece to a new PR since it doesn't have to do with venue/OSM work. I know it's just a small PR, but it just keeps things tight.